File: utilities/src/main/kotlin/lunalauf/rms/utilities/network/communication/message/response/RoundCountAcceptedResponse.java
package lunalauf.rms.utilities.network.communication.message.response;

import com.google.gson.annotations.SerializedName;
import lunalauf.rms.utilities.network.communication.message.type.ResponseType;

public class RoundCountAcceptedResponse extends RoundCountResponse {

    @SerializedName("rounds")
    public int newNumRounds;

    public RoundCountAcceptedResponse() {
        super(ResponseType.ROUND_COUNT_ACCEPTED);
    }

}

File: utilities/src/main/kotlin/lunalauf/rms/utilities/network/communication/message/response/RoundCountAcceptedResponse.kt
package lunalauf.rms.utilities.network.communication.message.response

import com.google.gson.annotations.SerializedName
import lunalauf.rms.utilities.network.communication.message.type.ResponseType

class RoundCountAcceptedResponse : RoundCountResponse(ResponseType.ROUND_COUNT_ACCEPTED) {
    @SerializedName("rounds")
    var newNumRounds = 0
}
